src/Chapter1.hs

```diff
@@ -653,7 +662,7 @@ You need to use recursion in this task. Feel free to return to it later, if you
 aren't ready for this boss yet!
 -}
 
-firstDigit n = error "firstDigit: Not implemented!"
+firstDigit n = if (abs n)<10 then (abs n) else firstDigit (div (abs n) 10)
```

Comment:
Great job 💯 

Here you can use a bit of alignment as well:

```suggestion
firstDigit n = 
    if  abs n < 10
    then abs n
    else firstDigit (div (abs n) 10)
```

Also note, that you are using `abs n` a lot in this function. You can move it into a separate variable in side of the `where` construction, for example 🙂